### PR TITLE
Update 1.0.1

### PR DESCRIPTION
**Pull request by SC426**
Jira Ticket: https://anaconda.atlassian.net/browse/PKG-596
Upstream Repo: https://github.com/wtforms/flask-wtf/tree/v1.0.1
Changelog: https://flask-wtf.readthedocs.io/en/1.0.x/changes/

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
```
Comment (katietz) [inline abs.yaml]: please remove this channel information from abs.yaml

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ff177185f891302dc253437fe63081e7a46a4e99aca61dfe086fb23e54fff2dc
+  url: https://files.pythonhosted.org/packages/d9/38/d4798dd05be711d666e1befb08b1d3db57bd146d1a14d15657f60c88b446/{{ name }}-{{ version }}.tar.gz
+  sha256: 34fe5c6fee0f69b50e30f81a3b7ea16aa1492a771fe9ad0974d164610c09a6c9
 
 build:
-  noarch: python
+  skip: True # [py<37]
```
Comment (katietz) [inline recipe/meta.yaml]: please make sure there are two spaces before the '#' sign, and one before the selector

**Review by katietz — CHANGES_REQUESTED** [review]
ok, with those 2 changes

**Review by varlackc — COMMENTED** [review]
while we no longer support versions older than `python 3.7` 
https://github.com/AnacondaRecipes/flask-wtf-feedstock/blob/6b2a94bb6f2bfac4f53336884206048d7238de22/recipe/meta.yaml#L13

Th upstream still supports `python` version `3.6` while using `3.7` is ok there might be some value on trying to stay as close as possible to the upstream,
However that point might be up to debate:
https://github.com/wtforms/flask-wtf/blob/main/setup.cfg#L36

python_requires = >= 3.6

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml]: Fixed. Please let me know if I've done this correctly now.

File: abs.yaml (hunk 1 above)
Comment (SC426) [reply abs.yaml]: Can I wait till upload to do this since I'm working on a dependency chain?

**Review by skupr-anaconda — APPROVED** [review]

File: abs.yaml (hunk 1 above)
Comment (ryanskeith) [reply abs.yaml]: Sure, just don't merge with an abs.yaml file.

**Comment by SC426** [comment]
> while we no longer support versions older than `python 3.7`
> 
> https://github.com/AnacondaRecipes/flask-wtf-feedstock/blob/6b2a94bb6f2bfac4f53336884206048d7238de22/recipe/meta.yaml#L13
> 
> Th upstream still supports `python` version `3.6` while using `3.7` is ok there might be some value on trying to stay as close as possible to the upstream, However that point might be up to debate: https://github.com/wtforms/flask-wtf/blob/main/setup.cfg#L36
> 
> python_requires = >= 3.6

I've made this change. Thank you for catching that.

**Review by AndrewVallette — APPROVED** [review]

**Pull request merged**